### PR TITLE
Tools: some test and update quadplane functions

### PR DESCRIPTION
**Pull request by khancyr**
The error was catch by @bnsgeyer !  The valid condition in arm_motors_with_rc_input was wrong, so the test was always pass and then disarm_motors_with_rc_input always pass too. 
This PR correct this problem.
It auto select the correct arming stick and put it in the good value.

### CODE REVIEW

File: Tools/autotest/common.py [hunk 1]
```diff
@@ -445,6 +445,39 @@ def set_throttle_zero(self):
...
+        """Set output to min with RC Radio taking into account REVERSED parameter"""
+        is_reversed = self.get_parameter("RC%u_REVERSED" % chan)
+        out_max = self.get_parameter("RC%u_MAX" % chan)
+        out_min = self.get_parameter("RC%u_MIN" % chan)
+        if is_reversed == 0:
+            self.set_rc(chan, out_min)
+        else:
+            self.set_rc(chan, out_max)
+
+    def get_arming_rudder(self):
```
Comment (peterbarker) [inline Tools/autotest/common.py]: This should simply return the value of `RCMAP_RUDDER`, shouldn't it?

Comment (khancyr) [reply Tools/autotest/common.py]: The issues is that RCMAP_RUDDER doesn't exist ... the channel is different for rover and plane/copter. Anyway I change to use the RCMAP !

File: Tools/autotest/common.py [hunk 2]
```diff
@@ -445,6 +445,39 @@ def set_throttle_zero(self):
...
+                                 mavutil.mavlink.MAV_TYPE_HELICOPTER,
+                                 mavutil.mavlink.MAV_TYPE_HEXAROTOR,
+                                 mavutil.mavlink.MAV_TYPE_OCTOROTOR,
+                                 mavutil.mavlink.MAV_TYPE_COAXIAL,
+                                 mavutil.mavlink.MAV_TYPE_TRICOPTER]:
+            return int(self.get_parameter("RCMAP_YAW"))
+        if self.mav.mav_type == mavutil.mavlink.MAV_TYPE_FIXED_WING:
+            return int(self.get_parameter("RCMAP_YAW"))
+        if self.mav.mav_type == mavutil.mavlink.MAV_TYPE_GROUND_ROVER:
+            return int(self.get_parameter("RCMAP_ROLL"))
```
[the remark this answers is not in the document]
Comment (khancyr) [reply Tools/autotest/common.py]: not supported by sub. I disable it!

**Comment by khancyr** [comment]
rebase and udpate some quadplane function that were failing

**Comment by peterbarker** [comment]
Merged this after renaming the get-rudder-channel function.

**Pull request closed without merging**